ballista/rust/core/src/serde/mod.rs

```diff
@@ -213,13 +213,23 @@ pub(crate) fn from_proto_binary_op(op: &str) -> Result<Operator, BallistaError>
         "Modulo" => Ok(Operator::Modulo),
         "Like" => Ok(Operator::Like),
         "NotLike" => Ok(Operator::NotLike),
+        "IsDistinctFrom" => Ok(Operator::IsDistinctFrom),
```

Comment:
Note that this method is copied from the private `from_proto_binary_op` method in DataFusion. It would be better to make that public rather than maintain a copy here.